quisp/modules/QRSA/HardwareMonitor/HardwareMonitor.h

```diff
@@ -90,7 +90,7 @@ class HardwareMonitor : public IHardwareMonitor {
   virtual void sendLinkTomographyRuleSet(int my_address, int partner_address, QNIC_type qnic_type, int qnic_index, unsigned long rule_id);
   virtual QNIC search_QNIC_from_Neighbor_QNode_address(int neighbor_address);
   virtual Matrix4cd reconstruct_Density_Matrix(int qnic_id);
-  virtual Matrix4cd extended_reconstruct_Density_Matrix(int qnic_id, int partner);
+  virtual Matrix4cd extended_reconstruct_density_matrix(int qnic_id, int partner);
```

Comment:
I said it should be `reconstruct_extended_density_matrix` or `extend_density_matrix` because methods should be started with verbs. "extended" is an adjective. this might confuse other developers.

Reply:
`Extended density matrix` would be a bit confusing as well. So, let me use `reconstruct_density_matrix`. The original `reconstruct_Density_Matrix` is no longer used.